CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 ## Unreleased
 
+## 1.18.0 / 2023-12-22
+
+* [FEATURE] promlint: Allow creation of custom metric validations. #1311
+* [FEATURE] api: Change HTTP Client into interface, so users can use customized clients, e.g. with retries on 429. #1387
```

Comment:
Thanks for putting it here -- I think it can be reverted 🙈 
Proposed: https://github.com/prometheus/client_golang/pull/1411